src/category_theory/abelian/basic.lean

```diff
@@ -152,83 +153,92 @@ is_iso_of_mono_of_strong_epi _
 end mono_epi_iso
 
 section factor
+local attribute [instance] non_preadditive_abelian
 
 variables {P Q : C} (f : P ⟶ Q)
 
+namespace images
+
 /-- The kernel of the cokernel of `f` is called the image of `f`. -/
 protected abbreviation image : C := kernel (cokernel.π f)
 
 /-- The inclusion of the image into the codomain. -/
-protected abbreviation image.ι : abelian.image f ⟶ Q :=
+protected abbreviation image.ι : images.image f ⟶ Q :=
```

Comment:
Aren't we inside the namespace? Why not just leave off `images.`?

Reply:
For whatever reason, if we leave it off, Lean thinks we're referring to `limits.image`.

Reply:
Ok!